modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -3183,6 +3171,8 @@ void ONNXImporter::parseQConv(LayerParams& layerParams, const opencv_onnx::NodeP
     Mat wt_sc = (w_scale.total() == outCn) ? w_scale : Mat(1, outCn, CV_32F, Scalar(w_scale.at<float>(0)));
 
     Mat out_sc = getBlob(node_proto, 6);
+    Mat out_zp = getBlob(node_proto, 7);
+    CV_Assert(out_sc.total() == 1 && out_zp.total() == 1);
```

Comment:
These are scalars per spec already, but I think we should add this assertion everywhere